src/router-view.ts

```diff
@@ -0,0 +1,236 @@
+import { Container } from 'aurelia-dependency-injection';
+import { createOverrideContext, OverrideContext } from 'aurelia-binding';
+// tslint:disable-next-line:max-line-length
+import { ViewSlot, ViewLocator, customElement, noView, BehaviorInstruction, bindable, CompositionTransaction, CompositionEngine, ShadowDOM, SwapStrategies, ResourceDescription, HtmlBehaviorResource, CompositionTransactionNotifier, View, CompositionTransactionOwnershipToken, Controller, ViewFactory, CompositionContext } from 'aurelia-templating';
+import {
+  Router,
+  ViewPortInstruction,
+  ViewPort
+} from 'aurelia-router';
+import { Origin } from 'aurelia-metadata';
+import { DOM } from 'aurelia-pal';
+
+class EmptyViewModel {
+
+}
+
+export class RouterView implements ViewPort {
+
+  static inject() {
+    return [DOM.Element, Container, ViewSlot, Router, ViewLocator, CompositionTransaction, CompositionEngine];
+  }
+
+  swapOrder: string;
+
+  layoutView: any;
+
+  layoutViewModel: any;
+
+  layoutModel: any;
```

Comment:
this is just an object, right?

Reply:
it is anything that we can use as model, similar to `model` in `<compose/>`